.github/workflows/build.yml

```diff
@@ -1,56 +1,29 @@
-# This is a basic workflow to help you get started with Actions
+# This yml is adapted from pull_request_test_build.yml
```

Comment:
Hey @wxharry, I think you can just describe what the yml file does in this comment line.